src/AutoRest.CSharp.V3/Plugins/Modeler.cs

```diff
@@ -190,6 +190,22 @@ private ConstantOrParameter[] ToParts(string httpRequestUri, Dictionary<string,
             return host.ToArray();
         }
 
+        private PathSegment[] ToPathParts(string httpRequestUri, Dictionary<string, ConstantOrParameter> parameters)
+        {
+            List<PathSegment> host = new List<PathSegment>();
+            foreach ((string text, bool isLiteral) in GetPathParts(httpRequestUri))
+            {
+                // WORKAROUND FOR https://github.com/Azure/autorest.modelerfour/issues/58
+                if (!parameters.ContainsKey(text))
+                {
+                    parameters[text] = StringConstant(text);
+                }
+                host.Add(new PathSegment(isLiteral ? StringConstant(text) : parameters[text], !isLiteral));
```

Comment:
This is duplicate code of the method above it with 1 line difference.

Reply:
Cleaned up.